{'station_locations': [[-0.5, 1.0, 0.15],
  [0.5, 1.25, 0.2],
  [1.0, 0.0, 0.3],
  [0.0, -0.75, 0.15]],
 'target_config': [[2, 1, 0], [1, 0, 1], [1, 1, 1], [1, 1, 0]],
 'robot_1_colors': [[1], [1], [0], [0]],
 'robot_2_colors': [[0], [0], [1], [1]]}